umongo/abstract.py

```diff
@@ -181,7 +184,7 @@ def _extract_marshmallow_field_params(self, mongo_world):
         params.update(self.metadata)
         return params
 
-    def as_marshmallow_field(self, params=None, mongo_world=False):
+    def as_marshmallow_field(self, params=None, mongo_world=False, **kwargs):
         """
         Return a pure-marshmallow version of this field.
 
```

Comment:
I don't think so.

`field_kwargs` are passed in `params`.

`kwargs` are schema-level parameters, sort of. The implementation I propose passes them to all fields although they are only needed in `EmbeddedField` (to be passed to nested schema). We could pass them to `EmbeddedField` only (and through `ListField`) but I figured we might as well pass `kwargs` to all fields. 
There could be other shcema-level parameters that would affect all `as_marshmallow_fields` and this would be the right place for them.